.rubocop.yml

```diff
@@ -0,0 +1,5 @@
+AllCops:
+  NewCops: enable
+  
+Metrics/BlockLength:
+  Max: 100
```

Comment:
I want to review the docs on this and make sure this is what we want.

Reply:
https://docs.rubocop.org/rubocop/configuration.html

Here's the link for reference. It has a section on IgnoredMethods which I originally typed up, and when I ran it Rubocop kicked back the same error for BlockLength. Inside of that error message it told me that IgnoredMethods was old syntac that had been replaced with AllowedMethods. I messed around with AllowedMethods for awhile before giving up and finding  rubocop --auto-gen-config. Everything for IgnoredMethods and the console command are near the bottom of the page.

Reply:
I think this

```
AllCops:
  NewCops: enable

Metrics/BlockLength:
  AllowedMethods: ["describe", "context"]
```
is more what we want because it doesn't impose the block length restriction just specifically on these method names, which are only used in specs.  If a method is too long in our actual code, we want to know about it and 100 is probably too many lines allowed in that case.